ocrd_tool.md

```diff
@@ -156,6 +156,7 @@ properties:
                 - recognition/font-identification
                 - recognition/post-correction
                 - layout/segmentation
+                - layout/segmentation/text-image
```

Comment:
OK, but some general thoughts:

I have not reviewed the DFKI method yet but in the future we may need sth more abstract than ``text-image`` and/or sth more specific to differentiate between various non-text regions. I am e.g. not certain how current OCR-D segmentation methods segment or classify tables, graphs, formulas? Is a comparable level of segmentation exposed by the other, i.e. Würzburg method?  

What about ``text-other`` ?